### PR TITLE
Add wide version of Tearsheet

### PR DESCRIPTION
**Pull request by moores2**
Contributes to #183 

Initial version of the wide Tearsheet

#### Changelog

A       packages/experimental/src/components/Tearsheet/Tearsheet.js
A       packages/experimental/src/components/Tearsheet/Tearsheet.stories.js
A       packages/experimental/src/components/Tearsheet/TearsheetShell.js
A       packages/experimental/src/components/Tearsheet/TearsheetShell.stories.js
A       packages/experimental/src/components/Tearsheet/_index.scss
A       packages/experimental/src/components/Tearsheet/_storybook-styles.scss
A       packages/experimental/src/components/Tearsheet/_tearsheet.scss
M       packages/experimental/src/components/_index.scss

**New**

- Wide Tearsheet

### CODE REVIEW

File: packages/experimental/src/components/Tearsheet/Tearsheet.js [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+  className: PropTypes.string,
+  /**
+   * The description for the close icon. Optional.
+   */
+  closeIconDescription: PropTypes.string,
+  /**
+   * Specifies the description of the Tearsheet.
+   * Optional.
+   */
+  description: PropTypes.node,
```
Comment (lee-chase) [inline packages/experimental/src/components/Tearsheet/Tearsheet.js:106]: Why is this not PropTypes.string?

**Review by lee-chase — APPROVED** [review]
1. Should the ghost button take up the full space available?
2. The tabs appear to be offset to the left by 16pxish
image
3. It appears to be 8, 10 and 10 pixels too short
image
4. Probably asked before but can we get storybook actions from the tabs

File: packages/experimental/src/components/Tearsheet/Tearsheet.js (hunk 1 above)
Comment (moores2) [reply packages/experimental/src/components/Tearsheet/Tearsheet.js:106]: So, that rich content can be provided if needed. It is standard practice for the ModalHeader etc to allow nodes for label, title etc, so is consistent with that.

**Comment by moores2** [comment]
> 1. Should the ghost button take up the full space available?
It only appears to. It's actually 25% of the width as per the other buttons and the spec. However, being a ghost the size is only apparent on hover.
> 2. The tabs appear to be offset to the left by 16pxish
>    image
Will fix.
> 3. It appears to be 8, 10 and 10 pixels too short
>    image
The heights of the individual elements in the style spec do not add up to the total shown. Since it is interim this will be deferred until it becomes final and we can review it properly.
> 4. Probably asked before but can we get storybook actions from the tabs
Will investigate.

**Pull request merged**